### PR TITLE
OCLOMRS-1046:Paging, searching and filtering through the list of concepts no longer works

### PR DESCRIPTION
**Pull request by jwnasambu**
# JIRA TICKET NAME:
[Paging through the list of concepts no longer works](https://issues.openmrs.org/browse/OCLOMRS-1046)

# Summary:
- This fixes pagination, filtering and searching through concepts.

### CODE REVIEW

**Comment by suruchee** [comment]
@jwnasambu , thanks for the PR but it's not loading any concepts at my end.

**Comment by jwnasambu** [comment]
@suruchee thanks for the feedback! Basing on the PR this is what I have  that I believe was supposed to be done basing on the ticket discription.

https://user-images.githubusercontent.com/33891016/133126150-3338d87a-3791-424b-9993-aab39e877569.mp4

File: src/apps/concepts/pages/ViewConceptsPage.tsx [hunk 1]
```diff
@@ -268,7 +268,23 @@ const ViewConceptsPage: React.FC<Props> = ({
     // i don't know how the comparison algorithm works, but for these arrays, it fails.
     // stringify the arrays to work around that
     // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [retrieveConcepts, retrieveDictionary, retrieveSource, containerUrl]);
+  }, [
+    retrieveConcepts,
```
Comment (HerbertYiga) [inline src/apps/concepts/pages/ViewConceptsPage.tsx:272]: cc @ibacher

**Comment by jwnasambu** [comment]
@suruchi you can try testing again.

**Review by hadijahkyampeire — APPROVED** [review]
Thanks for the fix @jwnasambu , this works fine, it looks like you and @suruchee have submitted the same work, we should merge one of the PRs and close the other.

**Comment by jwnasambu** [comment]
> Thanks for the fix @jwnasambu , this works fine, it looks like you and @suruchee have submitted the same work, we should merge one of the PRs and close the other.

Oh sorry! its unfortunate but I was the one who claimed the ticket and I was working on it.  It's not good to work on someone's ticket without there consent.

**Comment by hadijahkyampeire** [comment]
It's okay @jwnasambu it was a mistake with the reporting of the tickets, they made two different tickets but which were addressing the same issue, that's why you both had tickets that are different but solving the same issue, I have told @suruchee to rebase with your work and cover some other work in her PR since you have already addressed what she has in her PR.

**Pull request merged**